tests/CMakeLists.txt

```diff
@@ -331,22 +331,22 @@ add_test_case(h2_client_stream_create)
 add_test_case(h2_client_unactivated_stream_cleans_up)
 add_test_case(h2_client_connection_preface_sent)
 add_test_case(h2_client_ping_ack)
+add_test_case(h2_client_ping_ack_higher_priority)
 add_test_case(h2_client_setting_ack)
 add_test_case(h2_client_request_cookie_headers)
 add_test_case(h2_client_stream_complete)
 add_test_case(h2_client_close)
-#TODO add_test_case(h2_client_close_with_incomplete_stream)
 add_test_case(h2_client_stream_with_h1_request_message)
 add_test_case(h2_client_stream_err_malformed_header)
-#TODO add_test_case(h2_client_stream_err_state_forbids_frame)
+add_test_case(h2_client_stream_err_state_forbids_frame)
 add_test_case(h2_client_conn_err_stream_frames_received_for_idle_stream)
 add_test_case(h2_client_stream_ignores_some_frames_received_soon_after_closing)
 #TODO add_test_case(h2_client_conn_err_stream_frames_received_long_after_closing)
-#TODO add_test_case(h2_client_conn_err_stream_frames_received_after_rst_stream_received)
-#TODO add_test_case(h2_client_stream_receive_info_headers)
-#TODO add_test_case(h2_client_stream_err_receive_info_headers_after_main)
-#TODO add_test_case(h2_client_stream_receive_trailing_headers)
-#TODO add_test_case(h2_client_stream_err_receive_trailing_before_main)
+add_test_case(h2_client_conn_err_stream_frames_received_after_rst_stream_received)
```

Comment:
Oh, should this be a stream error instead of connection error? "An endpoint that receives any frame other than PRIORITY after receiving a RST_STREAM MUST treat that as a stream error (Section 5.4.2) of type STREAM_CLOSED" [stream_closed_state](https://httpwg.org/specs/rfc7540.html#StreamStates)